lib/approval/service.rb

```diff
@@ -1,4 +1,4 @@
-require 'approval-api-client-ruby'
+require 'insights-approval-api-client'
```

Comment:
Is this line needed since it is loaded from the Gemfile?

Reply:
@gmcculloug removed